modules/ROOT/pages/hosting-options.adoc

```diff
@@ -60,13 +60,13 @@ The expected output resembles:
 +
 `Name | examplelb`
 +
-. Get the CloudHub domain for the Anypoint DataGraph application:
+. Get the CloudHub domain for the Anypoint DataGraph app name:
 +
 `datagraph load-balancer-config describe`
 +
 The expected value resembles:
 +
-`Domain |  lb.anypointdns.net`
+`Domain |  datagraph-<org_id_excerpt>-<env_id_excerpt>`
```

Comment:
I would use a real value in the example:

`Domain |  datagraph-xz082edx-1585-901xe374-d0x0`